validation/validate_dc2_run2.1i_object_table.ipynb

```diff
@@ -0,0 +1,733 @@
+    "| South-West Corner | 49.917517    | -44.33        | 52.479       | -28.667       |\n",
+    "| South-East Corner | 73.794710    | -44.33        | 53.771       | -28.667       |\n",
+    "\n",
+    "(Note that the order of the rows above is different than in the DC2 papers.  The order of the rows above goes around the perimeter in order.)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "dc2_run2x_wfd = [[71.462228, -27.25], [52.250000, -27.25], [49.917517, -44.33], [73.794710, -44.33]]\n",
+    "dc2_run2x_ddf = [[53.764, -27.533], [52.486, -27.533], [52.479, -28.667], [53.771, -28.667]]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def plot_ra_dec(cat):\n",
+    "    \"\"\"We're just doing this on a rectilinear grid.\n",
+    "    We should do a projection, of course, but that distortion is minor in this space.\"\"\"\n",
+    "    fig = plt.figure(figsize=(8, 8))\n",
+    "    ax = plt.gca()\n",
+    "    ax.set_aspect(1)\n",
+    "\n",
+    "    plt.hist2d(cat['ra'], cat['dec'], bins=100)\n",
+    "    plt.xlim(plt.xlim()[::-1])  # Flip to East left\n",
+    "    plt.xlabel('RA [deg]')\n",
+    "    plt.ylabel('Dec [deg]')\n",
+    "    plt.colorbar(shrink=0.5, label='objects / bin')\n",
+    "\n",
+    "    # This region isn't quite a polygon.  The sides should be curved.\n",
+    "    wfd_region = Polygon(dc2_run2x_wfd, color='red', fill=False)\n",
+    "    ddf_region = Polygon(dc2_run2x_ddf, color='orange', fill=False)\n",
+    "    ax.add_patch(wfd_region)\n",
+    "    ax.add_patch(ddf_region)\n",
+    "    \n",
+    "    max_delta_ra = dc2_run2x_wfd[3][0] - dc2_run2x_wfd[2][0]\n",
+    "    delta_dec = dc2_run2x_wfd[1][1] - dc2_run2x_wfd[3][1]\n",
+    "    grow_buffer = 0.05\n",
+    "    ax.set_xlim(dc2_run2x_wfd[3][0] + max_delta_ra * grow_buffer,\n",
+    "                dc2_run2x_wfd[2][0] - max_delta_ra * grow_buffer)\n",
+    "    ax.set_ylim(dc2_run2x_wfd[3][1] - delta_dec * grow_buffer,\n",
+    "                dc2_run2x_wfd[1][1] + delta_dec * grow_buffer)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "plot_ra_dec(df)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The overall object density distribution looks good.\n",
+    "\n",
+    "Notes:\n",
+    "* We're missing the DDF region, which was specifically not included in this DR1 coadd.\n",
+    "* We're missing a bit of the South-West (lower-left, \"North Up, East Left\") corner.\n",
+    "* There are also a few patches that failed within the main region.\n",
+    "* There is an overall gradient N/S in object density, because we're plotting in rectilinear RA, Dec bins, which means that bins at the bottom in RA cover less area than those at the top.\n",
```

Comment:
If you plotted only stars/extendedness==0 would you see a gradient due to Galactic latitude, or is this too small a range to spot that?  If it is visible, might be a nice additional plot to make

Reply:
No, too small a range at too high above the Galactic plane to see a trend.